### DISCUSSION TITLE
Group 1 Project Discussion: Spatial Mismatch in the DMV

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#153] [Group Projects Feedback]
Hi Group 1! Great job writing this up and very interesting concept. Here is some high level feedback on the project pitch.

First, let's tackle a bit the question of "market failure" and "theory of change". This is always the hardest part! I am not sure I understand your idea of market failure here. There is certainly a _problem_, but I worry that this problem is reducible to "poor people are poor". In a working market, shouldn't it be the case that local salaries accurately account for the cost of living in proximity to jobs? And isn't it the case that people routinely move in order to have better access to things like jobs? In that case, something like a public transit project might just be a windfall (or free money) to people located near the access points -- it just makes them richer by giving them better access to cheap and efficient transit options. For a market failure, there would have to be some reason why these people are not able to efficiently relocate closer to better jobs. If you want to pursue, alternatively, the idea that companies use location to discriminate, that should be approached more explicitly. 

To be clear, I think you are on the right track overall, so this is not a criticism of your project choice, but you should think more carefully about the comparisons you want to draw so that you are not just comparing "people who get something" with "people who didn't get something". Think about inefficiencies or "stickinesses" in labor markets -- you might look at people who have reasons for being unable to move (working spouse, children in school) and look at responses like movement patterns or housing prices: These are the kinds of things that would equilibrate _in a working market_ but may be inefficiently priced here _because of market failure_. Does that make sense?

Next, let's talk about data. I suspect this will need a major rethink in the next version, as this data plan strikes me as infeasible. How are you going to convince firms to give you their (confidential) data on employee racial breakdown? How are you going to convince firms to respond to such a survey at all? Even if you were successful in this, what will you do with firms that open, close, or relocate between waves? I know we said that you do not necessarily need data that exists or that you will actually do, but it should be significantly more plausible than this. Thankfully, there are census-type data sources as you know, from the federal government and other routine surveys that may provide alternatives, particularly if you consider alternative outcomes.

This should also help you think more clearly about the comparison structure of your design. If you are using, for example, the PUMS data, you should expand more clearly in your next version on what your sample group is, who are treatment, and who are control. Again, you want to avoid simply comparing people who get access against those who don't -- you may want to compare across people, households, census tracts, etc, that receive the same level of access to the new transit options, _but_ that would have had different levels of market failure corresponding to different impacts. Again, good job overall and I think this is an interesting project that has a lot of room to grow!

**Comment by BeaLeydier** [comment]
Hi team!

Building off Ben's comments on your proposal. First of all, I love the topic, which is both quite original and very interesting, and I think you have a lot of good ideas for the design, but you need to pin down one direction and clarify your research question. Unfortunately, a lot of research is about having a very ambitious research agenda, and then having to narrow it down because we can't learn anything if we try to learn everything #deep. 

More specifically, I have noticed in your proposal a few things that are contradictory with each other, for example, you say 

> _spatial mismatch implies that companies that want to avoid hiring certain or all minority applicants may choose to establish their firms in publicly inaccessible locations_

 (which as Ben hinted, is a pretty strong claim that you would need to motivate further if that's the direction you decide to take your project in, and that can be an interesting claim to study per se)

but you also say 

> _we can reasonably assume that trends in employer hiring practices in the DMV area would be similar both before and after extension of the Purple Line into the suburbs of northern Virginia_

which is **literally** the opposite.

One issue here, which you are hinting at, is that a lot of things at play here are related to each other, or endogenous. For example, it is possible that people who live remotely have a harder time finding jobs because disconnected; it is also possible that people who are unemployed move to further locations because they are cheaper. It is also likely that when companies decide to establish themselves they anticipate or take into account the accessibility of the place (because companies plan things), as much as urban planners do the same kind of planning the other way around and try to anticipate where the needs will be (same).

So for your design to try and disentangle these things that are all related to each other, you can think if there can be some **exogenous** element to it; for example, if the metro line was built overnight, or if its map was drawn by some foreigners who took a map and drew a line (it has been done in the past), but this is not really the case here. In fact, the fact that the line has been delayed probably makes it even less exogenous, because residents and companies have known for a while about it, and are/will be able to adapt their behavior in anticipation of the line opening (think people buying houses in Crystal City after Amazon announced their HQ but before they moved). One thing you may try to explore, precisely because of the delay, is to study the anticipation effects vs the real effects : how much behaviors change because of the announcement of the line, how do behaviors change (people, companies moving), and whether these behavior changes in themselves carry any effect, over the actual usage of the line itself when it opens, for example.

Another thing to think about in your design is whether your T and C groups are comparable. You want them to be as comparable as possible; ideally you want them to be exactly the same in all aspects except the one aspect you are studying. But in your design, you make them very different by imposing T companies to be hiring Black employees and C companies to be hiring non Black employees, and I don't really understand why.

> The treatment groups (including pre and post policy implementation) will comprise of firms having recently hired Black employees both before and after the Purple Line extension project was completed. The control groups will consist of firms having recently hired non-Black employees both before and after the Purple Line project is completed.

To summarize, I think you will want to simplify your design and project by :
- Focusing on one outcome : whether it is unemployment rate, poverty, hiring decision and racial equity in companies, etc
- Focusing on one actor : are you interested in this outcome from the perspective of the people (eg employees, future employees) or of companies
- Outlining in what way your T and C group are different (ideally, only one way), and why **that** way is the critical thing that makes them not reach the **outcome** you focus on

Cheers!

**Comment by bbdaniels** [comment]
# Concept Note Review

The overall idea and contextual analysis remains solid, but the design of treatment/control groups have not improved. The "complications" section is, I think, much more accurate than the "measurement" section, which demonstrates that you are grappling with the fundamental problems in designing a "transit corridor"-type study. It is just not plausible that (a) the employment rate of Black people in ALL OF MARYLAND will be significantly impacted by this project, since you are basically looking at commuting between Bethesda, Silver Spring, UMD, and Takoma -- some of the least-Black parts of Maryland! (see map)

![Screen Shot](https://user-images.githubusercontent.com/1459122/227253262-fa0980d5-cf32-4c68-a631-3a600628184c.png)

Further, your data collection does not match up with your measurement and impact. You talk about employment rate as a primary outcome, but your data collection is firm-level surveying. You have also thrown in what feels like a bunch of random stuff from IPUMS without any careful reasoning as to why, when, or how these variables would be important. Are these outcomes? Channels? Confounders? The next version needs to pay very close attention to the design of the comparison (regression) along the lines we are building out in class. This whole section on data and estimation needs to align much more with your variables and outcomes of interest.

Let me make a concrete suggestion for you to focus around. Your key question is very obviously along the lines of "people who get access to metro get gains that others do not -- what are the size of those gains"? And I like this a lot! So let's get creative about it. Your ideal kind of "treatment" variable is "how close is the person to a new Metro station", so why not just use that as the treatment exposure and compare "across the gradient", as it were? You could, for example, grab home-sales records (these are public records) and filter for all addresses within, say, 10 miles of a PL stop. Then filter to sales before the announcement of the PL. Now you have a comparison sample -- locked-in homeowners close to, and far from, the new lines.

What next? Well, you could try to run a survey of these addresses by partnering with the state DOT or some such and giving nice vouchers to incentivize participation. You could do point-surveys at PL stops although this likely has low response and a lot of noise. You could get cold-room access to appropriate Federal (or state) data such as income tax returns for the addresses you have targeted, since homeowners = taxpayers. Then, you have the key ingredients for the measurement and analytical part of the study -- a sensible treatment range, a good comparator subset (locked-in homeowners), and excellent quality data and coverage.

Anyway -- take this forward. Be creative, and be specific, and I think you can have a very smart final project. Good work so far, keep it up!
